lib/active_merchant/billing/gateways/epay.rb

```diff
@@ -268,7 +279,7 @@ def xml_builder(params, soap_call)
 
       def authorize_post_data(params = {})
         params[:language] = '2'
-        params[:cms] = 'activemerchant'
+        params[:cms] = 'activemerchant_3ds'
```

Comment:
can we get the reasoning behind this change?

Reply:
Also, any chance we could support something like `application_id || 'activemerchant'`?https://github.com/activemerchant/active_merchant/blob/341b53eb0b3a129808389c5b4f7714b6a80d06cd/lib/active_merchant/billing/gateways/checkout_v2.rb#L98

Reply:
`application_id` could be set by a user of Active Merchant to identify their platform. With a static value all requests using active merchant are grouped on the same bucket. If you had support for `application_id`, you could differentiate between "shopping cart" A and B